opencensus/stats/measure_to_view_map.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.tags import tag_map
+from opencensus.stats.measurement import Measurement
+from opencensus.stats import measurement
+from opencensus.stats.view import View
+from opencensus.stats.view_data import ViewData
+from datetime import datetime
+from collections import defaultdict
+import logging
+
+
+class MeasureToViewMap(object):
+    """Measure To View Map stores a map from names of Measures to specific View Datas"""
+    def __init__(self):
+        self._map = defaultdict(list)  # stores the one-to-many mapping from Measures to View Datas
+        self._registered_views = {}  # stores a map from the registered View names to the Views
+        self._registered_measures = {}  # stores a map from the registered Measure names to the Measures
+        self._exported_views = set()  # stores the set of the exported views
+
+    @property
+    def exported_views(self):
+        """the current exported views"""
+        return self._exported_views
+
+    def get_view(self, view_name, timestamp):
+        """get the View Data from the given View name"""
+        view = self.get_view_data(view_name)
+        if view is None:
+            return None
+
+        view_data = ViewData(view=view, start_time=timestamp, end_time=timestamp)
```

Comment:
IIUC the variable `view` is a `ViewData` rather than a `View.` How do you set the rows/aggregation map for the `ViewData`?